ledger/sandbox-on-x/BUILD.bazel

```
@@ -451,43 +501,66 @@ server_conformance_test(
 
 conformance_test(
     name = "conformance-test-static-time",
+    hocon = True,
     ports = [6865],
     server = ":app",
-    server_args = [
-        "--contract-id-seeding=testing-weak",
-        "--participant participant-id=example,port=6865",
-        "--static-time",
-    ],
+    server_args = ["run"],
+    server_hocon_config = """
+    {
+      ledger.participants.default.api-server {
+        seeding=testing-weak
+        time-provider-type=static
+      }
+    }
+    """,
     test_tool_args = [
         "--verbose",
     ],
 )
 
 conformance_test(
     name = "conformance-test-in-memory-fan-out",
+    hocon = True,
     ports = [6865],
     server = ":app",
-    server_args = [
-        "--contract-id-seeding=testing-weak",
-        "--participant participant-id=example,port=6865",
-        "--buffered-ledger-api-streams",
-    ],
+    server_args = ["run"],
+    server_hocon_config = """
+    {
+      ledger.participants.default {
+        api-server {
+          seeding = testing-weak
+        }
+        index-service {
+          enable-in-memory-fan-out-for-ledger-api=true
+        }
+      }
+    }
+    """,
     test_tool_args = [
         "--verbose",
     ],
 )
 
 conformance_test(
     name = "conformance-test-in-memory-fan-out-tiny-buffer",
+    hocon = True,
     ports = [6865],
     server = ":app",
-    server_args = [
-        "--contract-id-seeding=testing-weak",
-        "--participant participant-id=example,port=6865",
-        "--ledger-api-transactions-buffer-max-size=3",
-        "--buffered-streams-page-size=1",
-        "--buffered-ledger-api-streams",
-    ],
+    server_args = ["run"],
+    server_hocon_config = """
+    {
+      ledger.participants.default {
+        api-server {
+          seeding = testing-weak
+        }
+        index-service {
+          enable-in-memory-fan-out-for-ledger-api=true
```

```suggestion
          enable-in-memory-fan-out-for-ledger-api = true
```
Nitpicking: Can we be consistent wrt to spaces everywhere around `=` in HOCON config sections?

---

we can try ;) Not sure it is realistic to enforce in each next PR then as its not autoformatted